buddyboss/buddyboss-platform
# PROD-883 Fix 1870 Condition added to prevent from duplicated first char with CJK IME

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1870.

### How to test the changes in this Pull Request:

1. Go to activity feed page and goto create a new feed editor area
2. Add IME language and switch to it like Korean Microsoft IME
3. Type any one letter and then second, you will notice first character is duplicated, but now will be resolved with this PR and no duplicate characters

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crQUlfV6d0n

### Other information:




### Changelog entry

> Condition added to prevent from duplicated first char with CJK IME
